std_msgs/Header header
geometry_msgs/Twist velocity
geometry_msgs/Point player
geometry_msgs/Point[] obstacles
geometry_msgs/Point[] goals
